src/index.js

```diff
@@ -114,6 +114,19 @@ export function releaseCapture(uri: string): void {
   }
 }
 
+export function captureScreenshot(
```

Comment:
can this be renamed to captureScreen maybe?

Reply:
Refactored